Tools/Scripts/webkitpy/port/win.py

```diff
@@ -59,12 +59,10 @@ class WinPort(ApplePort):
 
     VERSION_MIN = Version(5, 1)
     VERSION_MAX = Version(10)
-
     ARCHITECTURES = ['x86', 'x86_64']
-
     DEFAULT_ARCHITECTURE = 'x86_64'
-
     CRASH_LOG_PREFIX = "CrashLog"
+    DRIVER_NAMES = ('WebKitTestRunner', 'DumpRenderTree')
```

Comment:
Nit (and not really something I expect addressed here): does any current port on Windows support WK1? I thought with the death of Apple's Windows WebKit port that was gone?

Reply:
I don't believe they do, and I _think_ the entirety of the `win.py` file can be removed, but I didn't want to chase down that dragon right this moment.